frontend/src/styles.css

```
@@ -0,0 +1,233 @@
+  color-scheme: light;
+  --page-bg: #f4efe7;
+  --panel-bg: rgba(255, 252, 247, 0.76);
+  --panel-strong: rgba(255, 251, 245, 0.92);
+  --line-soft: rgba(22, 35, 61, 0.12);
+  --line-strong: rgba(22, 35, 61, 0.32);
+  --ink-strong: #16233d;
+  --ink-soft: #52607c;
+  --ink-muted: #73809b;
+  --accent: #b9522c;
+  --accent-soft: rgba(185, 82, 44, 0.14);
+  font-family: "Avenir Next", "Segoe UI", sans-serif;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)